src/components/app/MenuButtons/index.css

```diff
@@ -45,10 +17,24 @@
 }
 
 .menuButtonsButton:hover {
-  background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1));
+  background-color: rgba(0, 0, 0, 0.1);
 }
 
 .buttonWithPanel.open > .menuButtonsButton,
 .menuButtonsButton:hover:active {
-  background-image: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2));
+  background-color: rgba(0, 0, 0, 0.2);
+}
+
+.menuButtonsLink {
+  position: relative; /* This allows positioning the ::before element. */
+  cursor: default;
+  text-decoration: none;
+}
```

Comment:
I reused the class `menuButtonsButton` for the link too, and thats why it's much simpler now.